.gitignore

```diff
@@ -92,4 +92,18 @@ ENV/
 .idea/
 
 # mkdocs
-site
\ No newline at end of file
+site
+
+# vscode configuration files
+.vscode
+
+# python virtual environment using pythons's
+# virtualenv binary
+bin
```

Comment:
Not part of the[ official one ](https://github.com/github/gitignore/blob/master/Python.gitignore)